lib/org/openpsa/invoices/handler/invoice/crud.php

```diff
@@ -28,11 +28,12 @@ class org_openpsa_invoices_handler_invoice_crud extends midcom_baseclasses_compo
     public function _load_schemadb()
     {
         $this->_schemadb = midcom_helper_datamanager2_schema::load_database($this->_config->get('schemadb'));
-        if (   $this->_mode == 'create'
-            && count($this->_master->_handler['args']) == 1) {
+        if (   $this->_mode == 'create' && count($this->_master->_handler['args']) == 1)
+        {
```

Comment:
Coding style: bracket should be in the line above. Also, if you put multiple conditions in an `if` on the same line, there shouldn't be three whitespaces after the opening bracket